docs/.vitepress/theme/style.css

```diff
@@ -7,16 +7,25 @@
  * Fonts
  */
 
-@import url('https://fonts.googleapis.com/css2?family=Heebo:wght@400;600&display=swap');
-@import url('https://fonts.googleapis.com/css2?family=Cousine');
-@import url('https://fonts.googleapis.com/css2?family=Rubik:wght@600');
+/**
+ * @import url('https://fonts.googleapis.com/css2?family=Heebo:wght@400;600&display=swap');
+ * @import url('https://fonts.googleapis.com/css2?family=Cousine');
+ * @import url('https://fonts.googleapis.com/css2?family=Rubik:wght@600');
+ */
```

Comment:
Are these lines needed? Why not removing them?

Reply:
I kept them as a reference, but there seems no issue in removing them

Reply:
There is no need to keep commented code, especially if you're using **git** as it's a source version control and we can easily see the before/after and get to the removed code whenever needed (if needed) 😄